p2p/message_test.go

```diff
@@ -7,6 +7,54 @@ import (
 	"github.com/ChainSafe/gossamer/common"
 )
 
+func TestDecodeMessage(t *testing.T) {
+	genesisHash, err := common.HexToHash("0xdcd1346701ca8396496e52aa2785b1748deb6db09551b72159dcb3e08991025b")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	bestBlockHash, err := common.HexToHash("0x8dac4bd53582976cd2834b47d3c7b3a9c8c708db84b3bae145753547ec9ee4da")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	encStatus, err := common.HexToBytes("0x00020000000200000004c1e72400000000008dac4bd53582976cd2834b47d3c7b3a9c8c708db84b3bae145753547ec9ee4dadcd1346701ca8396496e52aa2785b1748deb6db09551b72159dcb3e08991025b0400")
```

Comment:
Where did you get his value from?

Reply:
from a message from the alexander testnet

Reply:
please document

Reply:
ok done

Reply:
There's also one in the next test

Reply:
ok added that also